.github/CHANGELOG.md

```diff
@@ -602,6 +570,38 @@ fully differentiable.
   array([1.0 - 2.0j])
   ```
 
+* The four-term parameter-shift rule, as used by the controlled rotation operations,
+  has been updated to use coefficients that minimize the variance as per
+  https://arxiv.org/abs/2104.05695.
+  [(#1206)](https://github.com/PennyLaneAI/pennylane/pull/1206)
+
+* A new transform `qml.transforms.invisible` has been added, to make it easier
+  to transform QNodes.
+  [(#1175)](https://github.com/PennyLaneAI/pennylane/pull/1175)
+
+  Marking a quantum function as invisible will inhibit any internal
+  quantum operation processing from being recorded by the QNode:
+
+  ```pycon
+  >>> @qml.transforms.invisible
```

Comment:
I'm afraid I'm not a fan of this implementation. It just feels like the wrong UI compared to everything else we've already established. Would recommend revisiting.

Reply:
:+1: